src/global/colors.js

```diff
@@ -112,7 +113,6 @@ export default {
 
   // FORMS
   buttonColor: '#1C497E',
-  disabledButtonColor: '#37393C',
```

Comment:
Are you sure this is not used? I see that is a duplicate, but wants to re-check that correct one is deleted.

Reply:
I have deleted the former one which is overridden by the latter one. I am sure that it would keep the color that is currently used.